### DISCUSSION TITLE
About issues and labels

### DISCUSSION THREAD

**Discussion by admin** [esm-tools/esm_tools#1011] [General]
Dear @esm-tools/all-users,

In order to give appropriate user support, we are also dependent on your feedback about ESM-Tools. That means, if you 
- encounter any wrong and/or unexpected behaviour of ESM-Tools; or 
- do not know how to solve any problem you are facing using ESM-Tools;

please let us know. 

You can report feedback and ask for support here on Github with respect to different kinds of problems that may arise:
- questions about the usage of ESM-Tools;
- request for a new feature that ESM-Tools does not support yet;
- report an error message that is raised during the usage of ESM-Tools;
- report an incorrect or unexpected behaviour indicating an error in the ESM-Tools source code.

It is possible to address these kinds of problems here on Github via the section 'Issues' and/or 'Discussions'. _**If you**_ 

- _**have a question related to the usage of ESM-Tools**_, please
-> open your question as a '**new discussion**' in the '**Discussions**' section (that you are reading right now).

- _**found a bug in the ESM-Tools source code, please**_:
-> open this as a '**new issue**' in the sections '**Issues**' and label it with the label '**bug**'.

- _**have a request for a new feature that ESM-Tools does not cover yet**_, please
-> also open this as a '**new issue**' in the sections '**Issues**' and label it with the label '**enhancements**'.

In general, we strongly recommend you to label your new issue or discussion entries. This is because certain kinds of issues and problems we consider having different priorities. Of high priority are for example bugs and certain missing information that keeps you from running your simulation.

Both under 'Issues' and 'Discussions' there are various labels that you can use to better describe your issues and questions. Some of them are highly recommended as they indicate a certain priority.
If you identify your issue to be one of the above described, please use one of the following labels to set the appropriate priority of your issue:

- A **question** (in Discussions) that is crucial to be able to continue work: -> label: **help** -> priority: **high**
- **Question** of less priority: -> label: **question** -> priority: **middle** to **low**
- Issue: **bug**: -> label: **bug** -> priority: **high**
- Issue: **enhancement** or **new feature**: -> label: **enhancement** -> priority: **middle** to **low**

Please note: All other labels that are available for issues and discussions can also help to describe your issue more resulting in a better support.

**Please also remember to close issues and discussions that are solved and answered for you.** 

If you have any further suggestions to this topic please write comments below.

Thank you!
Your @esm-tools/core-team

**Comment by nwieters** [comment]
Dear @esm-tools/all-users,

we have integrated an **automated workflow** in Github regarding issues and pull requests:
- Issues and pull requests that have been **inactive for 365 days** will be labelled with the label `Stale`
- A comment will be added to these issues and PRs, briefly describing this workflow
- _Inactive means:_ the issue or PR **has not been updated and no comment has been added within the last 365 days**.
- **If no update or new comment will be added within the following 30 days**, issues that are labelled as Stale **will be closed**. PR will not be closed.
- This workflow will not be applied on issues and PR labelled as `enhancement` or `feature request`
- This **workflow can be reset by updating the issue/PR or adding a new comment**. The day counter will be reset and the label `Stale` will be removed automatically.
